manifests/03_operator.cr.yaml

```diff
@@ -6,3 +6,5 @@ metadata:
     release.openshift.io/create-only: "true"
 spec:
   managementState: Managed
+  # Configure the log level of managed deployments
+  logLevel: "4"
```

Comment:
> With the change from previous comment, you should not need this, I hope. However, it changes the default loglevel to 2.

Done
 
> Isn't it weird we're running with debug loglevel by default?

I guess so. I wasn't intending to affect the logging behavior in this PR, not sure why previous maintainers decided to set 4 as the default.

Reply:
I've updated the verbosity of logging of rotation initiation to `2`, since it is infrequent and is something that the user should be informed of even without debug logging enabled.